build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in

```diff
@@ -82,6 +82,89 @@
     <content_attribute id="money-purchasing">mild</content_attribute>
   </content_rating>
   <releases>
+    <release date="2019-12-04" version="3.3.4">
+      <url>https://musescore.org/en/3.3.4</url>
+      <description>
+        <p>Fixes</p>
+        <ul>
+          <li>Palette names were scrambled and nearly impossible to read (Windows 7)</li>
+          <li>Scale of palette was incorrect for high and low DPI displays</li>
+          <li>Courtesy accidentals disappeared after an octave change using Ctrl(Cmd)+Up/Down</li>
+          <li>Crash reports could not be sent</li>
+        </ul>
+      </description>
+    </release>
+    <release date="2019-11-26" version="3.3.3">
+      <url>https://musescore.org/en/3.3.3</url>
+      <description>
+        <p>Improvements:</p>
+        <ul>
+          <li>Tie button behaves as a toggle now</li>
+          <li>Enable changing notes duration if selecting note stem, hook or a range</li>
+          <li>Visual improvements for palettes</li>
+          <li>Mark notes that are out of instrument range with red or yellow color even when they are selected</li>
+          <li>Save Online is now fully synchronised with the musescore.com upload page</li>
+        </ul>
+        <p>Fixes:</p>
+        <ul>
+          <li>Export Pdf on Mac with Muse Jazz Text was garbled and unreadable if no printers were setup on a machine</li>
+          <li>Entering notes with mouse failed in Italian TAB</li>
+          <li>Scoreview jumped back to start of score during note input on (auto)save</li>
+          <li>Sticking could not be copy-pasted</li>
+          <li>Extensions could not be installed on macOS Catalina</li>
+        </ul>
+      </description>
+    </release>
+    <release date="2019-11-14" version="3.3.2">
+      <url>https://musescore.org/en/3.3.2</url>
+      <description>
+        <p>Fixes</p>
+        <ul>
+          <li>"Save online" failed in some cases</li>
+          <li>Palettes were incorrectly placed when using multiple HighDPI monitors and scaling</li>
+          <li>Palettes disappeared on Ubuntu 18.04 in some cases</li>
+        </ul>
+      </description>
+    </release>
+    <release date="2019-11-13" version="3.3.1">
+      <url>https://musescore.org/en/3.3.1</url>
+    </release>
```

Comment:
I think this is probably not needed, 3.3.1 was drawn back and that link doesn't work anymore.

Also please sign the CLA, the link is in the description of the PR ;-)

Reply:
done